packages/plugins/model-train/cycle-gan-model-train/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Change Log
```

Comment:
Hmm..., I don't think we need to name it as cycle-gan-model-train :(

Reply:
How about cycle-gan-tensorflow-model-train?